src/main/java/com/ombremoon/enderring/common/init/entity/StatusEffectInit.java

```diff
@@ -50,25 +50,26 @@ public class StatusEffectInit {
                     .addAttribute(new AttributeModifier(
                                     UUID.fromString("b9f71f4c-cc64-4b36-b086-7b8bdb40834"),
                                     "CRIMSONSPILL_CRYSTAL",
-                                    0.6F, AttributeModifier.Operation.ADDITION),
+                                    30F, AttributeModifier.Operation.ADDITION),
                             ()-> Attributes.MAX_HEALTH
                     ).build());
 
+
     public static final RegistryObject<MobEffect> CRIMSONBURST_CRYSTAL = registerSimpleEffect("crimsonburst_crystal_tear",
             new EffectBuilder(EffectType.HEALTH_REGEN)
                     .addAttribute(new AttributeModifier(
                                     UUID.fromString("f9f71f34-cc64-64b3-6b08-67b8bdb40834"),
                                     "CRIMSONBURST_CRYSTAL",
-                                    0.6F, AttributeModifier.Operation.ADDITION),
+                                    20F, AttributeModifier.Operation.ADDITION),
```

Comment:
should be doing health regen not max hp https://eldenring.wiki.fextralife.com/Crimsonburst+Crystal+Tear

Reply:
kk